PSI_Project/Services/GoalService.cs

```diff
@@ -0,0 +1,37 @@
+﻿using PSI_Project.Repositories;
```

Comment:
Why is this file needed? Do you plan to put some logic in here in the future? Cause right now method distribution in GoalServices and GoalRepository is the same as we had in smthDatabaseOperations and smthHandler, which wasn't a good solution